code/modules/research/designs/weapon_designs.dm

```diff
@@ -18,7 +18,7 @@
 	name = "Decloner"
 	desc = "Your opponent will bubble into a messy pile of goop."
 	id = "decloner"
-	req_tech = list("combat" = 6, "materials" = 7, "biotech" = 5, "powerstorage" = 6)
+	req_tech = list("combat" = 8, "materials" = 7, "biotech" = 5, "powerstorage" = 6)
 	build_type = PROTOLATHE
```

Comment:
Yeah, exactly. The only thing harder to get is the teleport exosuit module, which Im pretty sure youd have better chances of winning the lottery than getting a BS 9 item from the experimentor.